docs/about/release-notes.rst

```diff
@@ -28,6 +28,7 @@ Bugfixes
 ~~~~~~~~
 
 * Fix coordinate and attribute comparisons to treat NaN (not-a-number) values as equal, which previously prevented most operations with data arrays or datasets that contained NaN values in their coordinates or attributes `#2331 <https://github.com/scipp/scipp/pull/2331>`_.
+* Fix bug where inputs of ``transform_coords`` were not deleted when ``keep_inputs=False`` `#2349 <https://github.com/scipp/scipp/pull/2349>`_.
```

Comment:
Has this actually been released yet?

Reply:
Oh, you are right. I thought this was part of 0.10